docs/schema/2.0/manifest.schema.json

```diff
@@ -75,7 +75,9 @@
         },
         "identifier": {
           "type": "string",
-          "maxLength": 128
+          "minLength": 6,
+          "maxLength": 128,
+          "pattern": "^[a-zA-Z0-9_-]+$"
```

Comment:
I think we should also allow dot char.
AppIds like 'com.example.myapp" are common in Apple and Google stores

Reply:
ok, makes sense

Reply:
added